### ISSUE TITLE
Bad results for some picture types

### ISSUE THREAD

**Issue by QuantScientist** [jantic/DeOldify#105]
Dear Team,
I wonder what could be causing this (bad one is the photo on the bottom). 
I used colorize_gen_192.h5 in conjunction with a render_factor of 41. 

![de jpg](https://user-images.githubusercontent.com/18743986/57851697-e71c7d00-77e9-11e9-91c2-14164d8fcf85.png)

Thanks,

**Comment by jantic** [comment]
There really isn't enough information to go off of here to really help you.  Mainly, I can't reproduce the results for any version of the image I found online (big or small).  So a direct url to the source would help in that. Here's what I get for 41 using https://www.ceskaordinace.cz/obrazek-text-6742-310-min.jpg as the source:

![41](https://user-images.githubusercontent.com/179759/57889689-5003f500-77ea-11e9-836e-47778aaf5fd4.png)

And with a more carefully selected render_factor of 14, derived from using the code under "See how well render_factor values perform on the image here" in the Colab:

![14](https://user-images.githubusercontent.com/179759/57889693-54301280-77ea-11e9-8201-540cb75a15ed.png)

That being said, I can tell you a few things:
1.  jpeg artifacts have a negative impact on performance for sure.  Looks like I'm seeing that here.
2.  render_factor is important at this stage of research (still in development!).  At this point it's a matter of trial and error (with that tool I described above).  Should it be this way?  No.  But I don't have easy answers on how to do away with that yet.  It probably has a lot to do with training on small square image sizes and not generalizing well in the size dimensions, but the hard part is figuring out how to adequately solve that.
